python/gtsam/examples/TranslationAveragingExample.py

```diff
@@ -0,0 +1,112 @@
+"""
+GTSAM Copyright 2010-2018, Georgia Tech Research Corporation,
+Atlanta, Georgia 30332-0415
+All Rights Reserved
+Authors: Frank Dellaert, et al. (see THANKS for the full author list)
+
+See LICENSE for the license information
+
+This example shows how 1dsfm uses outlier rejection (MFAS) and optimization (translation recovery)
+together for estimating global translations from relative translation directions and global rotations.
+The purpose of this example is to illustrate the connection between these two classes using a small SfM dataset.
+
+Author: Akshay Krishnan
+Date: September 2020
+"""
+
+import numpy as np
+
+import gtsam
+from gtsam.examples import SFMdata
+
+
+def get_data():
+    """"Returns data from SfMData.createPoses(). This contains global rotations and unit translations directions."""
+    # Using toy dataset in SfMdata for example.
+    poses = SFMdata.createPoses(gtsam.Cal3_S2(50.0, 50.0, 0.0, 50.0, 50.0))
+    rotations = gtsam.Values()
+    translation_directions = []
+    for i in range(0, len(poses) - 2):
+        # Add the rotation
+        rotations.insert(i, poses[i].rotation())
```

Comment:
would be nice to clarify what rotations represents: is it `wRc` or `cRw` here?

Reply:
Have clarified in the comment above.

Reply:
Thanks Akshay. I think it would be even more readable if we did it in the code itself. What do you think about the following:

```python
    # Using toy dataset in SfMdata for example.
    wTc_list = SFMdata.createPoses(gtsam.Cal3_S2(50.0, 50.0, 0.0, 50.0, 50.0))
    # Rotations of the cameras in the world frame
    wRc_list = gtsam.Values()
    # Normalized translation directions for pairs of cameras - from camera i to camera j,
    # in the coordinate frame of the first camera.
    jti_directions = []
    for i in range(0, len(poses) - 2):
        # Add the rotation.
        wRc_list.insert(i, wTc_list[i].rotation())
```

Reply:
Can I use `wRc_values` instead of `wRc_list`? Because it is not really a list. 
Also, I think direction from i to j should be `itj_direction` since i is the reference frame? 
And if this direction is expressed in the world frame, can I call it `w_itj_direction`?

Reply:
Hmm i'm a bit confused here. The relative direction in the world frame is just temporary (for the purpose of generating actual relative translation directions, in a camera's own frame), right?

Because the E matrix will only give us in one's own frame.

Reply:
if so, i’m good with any of your suggestions about the names for it